api/src/main/java/org/pmiops/workbench/actionaudit/auditors/EgressEventAuditorImpl.kt

```diff
@@ -32,15 +32,15 @@ constructor(
 ) : EgressEventAuditor {
 
     /**
-     * Converts the AoU-chosen cluster name into the actual VM name as reported by Google Cloud's
+     * Converts the AoU-chosen runtime name into the actual VM name as reported by Google Cloud's
      * flow logs. Empirically, an "-m" suffix is added to the VM name, due to Leo team's use
      * of Dataproc (see https://cloud.google.com/dataproc/docs/concepts/configuring-clusters/metadata).
      *
      * Note: this pattern may change if the Leo team switches to using raw VMs instead of Dataproc
```

Comment:
No action today, but a reminder for this part

Reply:
Thanks for pointing this out - I was not tracking this. I've added a reference to this ticket, and cross-referenced in the AC there: https://precisionmedicineinitiative.atlassian.net/browse/RW-4848 .